### PR TITLE
Update CHANGELOG.md for v1.5.0

### PR DESCRIPTION
**Pull request by stevedomin**
This change in ExUnit was missed in the release changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -339,6 +339,10 @@ This release brings further improvements to Calendar types. It adds arithmetic a
 
   * [EEx] Deprecate `<%= ` in "middle" and "end" expressions, e.g.: `<%= else %>` and `<%= end %>`
 
+#### ExUnit
```
Comment (whatyouhide) [inline CHANGELOG.md]: I personally think of this as a bug fix more than a deprecation though 😕

Comment (stevedomin) [inline CHANGELOG.md]: Happy to change it. It didn't really like a bug for me, that's why I used deprecation. As in, it wasn't entirely clear to me that it was the behaviour people expected (the impossibility of having multiple `describe` blocks with the same name).

Comment (josevalim) [inline CHANGELOG.md]: It is a bug fixes because otherwise the setup blocks would all get mixed up. :)

Comment (stevedomin) [inline CHANGELOG.md]: That makes sense! Changed.

**Comment by lexmag** [comment]
Thank you @stevedomin.

**Pull request merged**